src/lib/FreeC/Backend/Coq/Converter/Expr.hs

```diff
@@ -210,14 +212,13 @@ convertExpr' (IR.ErrorExpr srcSpan msg _) typeArgs [] = do
     ++ "Expected 1 type arguments, got "
     ++ show (length typeArgs)
     ++ "."
-  let partialArg = Coq.Qualid (fst Coq.Base.partialArg)
   typeArgs' <- mapM convertType' typeArgs
-  return
-    (genericApply Coq.Base.partialError
-                  [partialArg]
-                  typeArgs'
-                  [Coq.InScope (Coq.string msg) Coq.Base.stringScope]
-    )
+  args'     <- mapM convertExpr args
+  let partialArg = [Coq.Qualid (fst Coq.Base.partialArg)]
+      callee     = genericApply Coq.Base.partialError partialArg typeArgs' []
+  generateApplyN (1 :: Int)
```

Comment:
Is the type annotation really needed?

```suggestion
  generateApplyN 1
```

Reply:
No, but not type-annotating an Integer gave me a warning once while I tried different solutions for this issue. But this type annotation isn't needed here because the type annotation of `generateApplyN` specifies the first argument to be of type `Int`.